frontend/packages/dev-console/src/components/topology/shapes/Badge.scss

```
@@ -0,0 +1,11 @@
+.odc-badge-icon {
```

Convention would be:
```suggestion
.odc-badge {
```